CONTRIBUTOR_LADDER.md

```diff
@@ -41,17 +41,48 @@ Description: A Contributor contributes directly to the project and adds value to
     * Promotes the project in public
 * Privileges:
```

Comment:
since the other sections also have a "Process of becoming a ___" section, and this is not the first section, I think it would help to just add a line about the process simply being a community member who submits PRs, and that we don't have a nomination process for this one